docs/src/SUMMARY.md

```diff
@@ -1,38 +1,42 @@
-# Summary
+[Introduction](./introduction.md)
+[Installation](./installation.md)
+[FAQ](./faq.md)
 
-- [Introduction](./introduction.md)
-- [Quickstart](./quickstart.md)
-    - [Netcat](./quickstarts/netcat.md)
-    - [Agones + Xonotic (Sidecar)](./quickstarts/agones-xonotic-sidecar.md)
-    - [Agones + Xonotic (xDS)](./quickstarts/agones-xonotic-xds.md)
-- [Usage](./using.md)
-  - [File Configuration](./file-configuration.md)
-- [Integrations](./integrations.md)
-- [Proxy Mode](./proxy-mode.md)
-  - [Concepts](proxy/concepts.md)
-  - [Filters](./filters.md)
-      - [Capture](proxy/filters/capture.md)
-      - [Compress](proxy/filters/compress.md)
-      - [Concatenate Bytes](proxy/filters/concatenate_bytes.md)
-      - [Debug](proxy/filters/debug.md)
-      - [Drop](proxy/filters/drop.md)
-      - [Firewall](proxy/filters/firewall.md)
-      - [Load Balancer](proxy/filters/load_balancer.md)
-      - [Local Rate Limit](proxy/filters/local_rate_limit.md)
-      - [Match](proxy/filters/match.md)
-      - [Pass](proxy/filters/pass.md)
-      - [Timestamp](proxy/filters/timestamp.md)
-      - [Token Router](proxy/filters/token_router.md)
-      - [Writing Custom Filters](proxy/filters/writing_custom_filters.md)
-  - [Metrics](proxy/metrics.md)
-  - [Maxmind](proxy/maxmind.md)
-- [xDS Management API](./xds.md)
-  - [Built-in xDS Providers](./xds/builtin.md)
-    - [Filesytem](./xds/providers/filesystem.md)
-    - [Agones](./xds/providers/agones.md)
-  - [Metrics](./xds/metrics.md)
-- [Administration](./admin.md)
-- [Client SDKs](./sdks.md)
-    - [Unreal Engine](./sdks/unreal-engine.md)
-- [Examples](./examples.md)
-- [FAQ](./faq.md)
+# Services
+- [Proxy](./services/proxy.md)
+    - [Filters](./services/proxy/filters.md)
+        - [Capture](./services/proxy/filters/capture.md)
+        - [Compress](./services/proxy/filters/compress.md)
+        - [Concatenate Bytes](./services/proxy/filters/concatenate_bytes.md)
+        - [Debug](./services/proxy/filters/debug.md)
+        - [Drop](./services/proxy/filters/drop.md)
+        - [Firewall](./services/proxy/filters/firewall.md)
+        - [Load Balancer](./services/proxy/filters/load_balancer.md)
+        - [Local Rate Limit](./services/proxy/filters/local_rate_limit.md)
+        - [Match](./services/proxy/filters/match.md)
+        - [Pass](./services/proxy/filters/pass.md)
+        - [Timestamp](./services/proxy/filters/timestamp.md)
+        - [Token Router](./services/proxy/filters/token_router.md)
+        - [Writing Custom Filters](./services/proxy/filters/writing_custom_filters.md)
+    - [Metrics](./services/proxy/metrics.md)
+
+---
```

Comment:
Well the intention was to add more breaks as we add more services to give them a bit more spacing since they can have a lot of parts to them. Here's how it would look like with a third service chapter. I don't feel super strongly about it one way or another.

## Without Break

<img width="284" alt="Screenshot 2023-01-25 at 11 24 47" src="https://user-images.githubusercontent.com/4464295/214541522-5321db0d-5a75-4655-b7cf-252f04261873.png">


## With Break

<img width="284" alt="Screenshot 2023-01-25 at 11 35 42" src="https://user-images.githubusercontent.com/4464295/214541543-20678207-1d65-4b67-8a0d-46935c8daefe.png">

Reply:
Ah that makes perfect sense. So don't mind if you leave it in, or remove it and put it back later -- whatever works 👍🏻